cirq-core/cirq/ops/linear_combinations.py

```diff
@@ -459,6 +476,21 @@ def qubits(self) -> Tuple[raw_types.Qid, ...]:
         return tuple(sorted(qs))
```

Comment:
We should document the `qubits` method to say that we are always returning a sorted list of qubits which are used in the pauli sum.

Reply:
Done.